package.json

```diff
@@ -8,9 +8,10 @@
     "url": "git+https://github.com/gridstack/gridstack.js.git"
   },
   "scripts": {
-    "build": "grunt ; doctoc ./README.md ; doctoc ./doc/README.md ; doctoc ./doc/CHANGES.md",
+    "build": "mkdir dist && grunt && doctoc ./README.md && doctoc ./doc/README.md && doctoc ./doc/CHANGES.md",
```

Comment:
&& are needed to run on windows instead of ;

Reply:
`mkdir dist` will fail if mkdir exists. You can pass `mkdir -p` to only create it if it does not exist

Reply:
looks like `-p` is only supported on *nix versions of the utility